packages/sandcastle/src/SandcastleDialog.css

```diff
@@ -0,0 +1,38 @@
+.sc-dialog {
+  --inset: var(--stratakit-space-x3);
+  position: fixed;
+  inset: var(--inset);
+  z-index: 50;
+  margin: auto;
+  display: flex;
+  height: fit-content;
+  max-height: calc(100dvh - var(--inset) * 2);
+  flex-direction: column;
+  gap: var(--stratakit-space-x4);
+  overflow: auto;
+  border-radius: var(--stratakit-ext-radius-lg);
+  background-color: var(--stratakit-color-bg-page-base);
+  padding: var(--stratakit-space-x4);
+  box-shadow: var(--stratakit-shadow-dialog-base);
+
+  .sc-dialog-footer {
+    align-self: flex-end;
+    margin-top: var(--stratakit-space-x4);
+  }
+
+  @media (min-width: 500px) {
```

Comment:
The modal looks good on mobile and responsive to page resizes. The main page (viewer and gallery/editor) do not seem optimized for smaller screens yet, is that coming in a future PR?

Reply:
Longer term we've discussed making sandcastle "look nice" on mobile even if it's not very functional. However this is not a focus for the first version.
These styles were based on the example from the [AriaKit Dialog](https://ariakit.org/components/dialog) sample code which already included this so I just kept it. It's especially good because this is a very generic component so when we do take a pass for mobile styles this is already ready to go.